inc/az_ulib_ipc_api.h

```diff
@@ -245,7 +195,8 @@ AZ_NODISCARD az_result az_ulib_ipc_unpublish(
  * need this interface anymore, it shall release the handle by calling
  * az_ulib_ipc_release_interface().
  *
- * @note    **Do not release an interface will cause memory leak.**
+ * @note    **Do not release an interface will cause memory leak and block the interface to be
```

Comment:
nit: "Not releasing an interface will cause a memory leak and will block the interface from being properly released."

Reply:
Not sure what you meant by "an block the interface to be released" as I think that's self-explanatory from mentioning that you must release the interface.